### ISSUE TITLE
bbb-dl install problems

### ISSUE THREAD

**Issue by flightcomp** [C0D3D3V/bbb-dl#47]
I installed python3.8, ffmpeg and bbb-dl:
python3.8 -m pip install --user bbb-dl
without problems but when I execute:
bbb-dl -h
I get the following result:

Traceback (most recent call last):
  File "/home/roberto/.local/bin/bbb-dl", line 5, in <module>
    from bbb_dl.main import main
  File "/home/roberto/.local/lib/python3.8/site-packages/bbb_dl/main.py", line 31, in <module>
    from bbb_dl.ffmpeg import FFMPEG
  File "/home/roberto/.local/lib/python3.8/site-packages/bbb_dl/ffmpeg.py", line 13, in <module>
    from bbb_dl.utils import PathTools as PT, Log, formatSeconds
  File "/home/roberto/.local/lib/python3.8/site-packages/bbb_dl/utils.py", line 23, in <module>
    from PySide6.QtCore import QDateTime
ModuleNotFoundError: No module named 'PySide6'
Error in sys.excepthook:
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/apport_python_hook.py", line 63, in apport_excepthook
    from apport.fileutils import likely_packaged, get_recent_crashes
  File "/usr/lib/python3/dist-packages/apport/__init__.py", line 5, in <module>
    from apport.report import Report
  File "/usr/lib/python3/dist-packages/apport/report.py", line 30, in <module>
    import apport.fileutils
  File "/usr/lib/python3/dist-packages/apport/fileutils.py", line 23, in <module>
    from apport.packaging_impl import impl as packaging
  File "/usr/lib/python3/dist-packages/apport/packaging_impl.py", line 24, in <module>
    import apt
  File "/usr/lib/python3/dist-packages/apt/__init__.py", line 23, in <module>
    import apt_pkg
ModuleNotFoundError: No module named 'apt_pkg'

Original exception was:
Traceback (most recent call last):
  File "/home/roberto/.local/bin/bbb-dl", line 5, in <module>
    from bbb_dl.main import main
  File "/home/roberto/.local/lib/python3.8/site-packages/bbb_dl/main.py", line 31, in <module>
    from bbb_dl.ffmpeg import FFMPEG
  File "/home/roberto/.local/lib/python3.8/site-packages/bbb_dl/ffmpeg.py", line 13, in <module>
    from bbb_dl.utils import PathTools as PT, Log, formatSeconds
  File "/home/roberto/.local/lib/python3.8/site-packages/bbb_dl/utils.py", line 23, in <module>
    from PySide6.QtCore import QDateTime
ModuleNotFoundError: No module named 'PySide6'

**Comment by C0D3D3V** [comment]
what is the output if you try to install PySide6 using pip

what operating system are you using?

**Comment by flightcomp** [comment]
I'm using Ubuntu 18.04. Should I have to install qt6 before installing pyside6? Ubuntu 18.04 isn't supported development host for Qt6, see https://www.qt.io/blog/qt6-development-hosts-and-targets.

**Comment by C0D3D3V** [comment]
At least I think pside6 does not depend on qt6, It can also use qt5. But I will remove pyside6 in a minute from this project, and then do a new release...

**Comment by C0D3D3V** [comment]
Please try v1.0.6. Open a new issue if you still have problems.
